cmd/tailscale/cli/web.go

```diff
@@ -43,11 +43,18 @@ var webCSS string
 //go:embed auth-redirect.html
 var authenticationRedirectHTML string
 
-var tmpl *template.Template
+//go:embed syno-token-redirect.html
+var synoTokenRedirectHTML string
+
+var (
+	webTmpl               *template.Template
+	synoTokenRedirectTmpl *template.Template
+)
 
 func init() {
```

Comment:
can we not do this parsing work until the "web" command starts?

it's a little silly for "tailscale status" to do all this work.